File: Python/SivitasAkademik.py
# Import parent
from Human import Human

class SivitasAkademik(Human) :
    # Variabel milik kelas ini
    _asalUniv = ''
    _emailEdu = ''
    
    # Konstruktor
    def __init__(self, nik, nama, jenis_kelamin, asalUniv, emailEdu):
        # Mengakses variabel yang terdapat pada parent nya
        super().__init__(nik, nama, jenis_kelamin)
        # ini milik sendiri
        self._asalUniv = asalUniv
        self._emailEdu = emailEdu
    
    # Getter
    def get_asalUniv(self) :
        return self._asalUniv
    def get_emailEdu(self) :
        return self._emailEdu
    
    # Setter
    def set_asalUniv(self, asalUniv):
        self._asalUniv = asalUniv
    def set_emailEdu(self, emailEdu):
        self._emailEdu = emailEdu

File: C++/SivitasAkademik.cpp
// include parentnya yakni Human
// #include "Human.cpp"
using namespace std;

// Sivitas akademik sebagai child, Human sebagai parent
class SivitasAkademik : public Human{
    // buat variabel untuk kelas ini
    protected:
        string asal_universitas;
        string email_edu;

    public:
        // konstruktor
        // SivitasAkademik(){}
        // disini dilakukan pengambilan konstruktor dari parent
        SivitasAkademik(string nama, string nik, string jenis_kelamin, string asal_universitas, string email_edu) : Human(nama, nik, jenis_kelamin) {
            this->asal_universitas = asal_universitas;
            this->email_edu = email_edu;
        }

        // Getter dan setter
        // Getter
        string getAsalUniv() {
            return this->asal_universitas;
        }
        string getEmailEdu() {
            return this->email_edu;
        }

        // Setter
        void setAsalUniv(string asal_universitas) {
            this->asal_universitas = asal_universitas;
        }
        void setEmailEdu(string email_edu) {
            this->email_edu = email_edu;
        }
        
        // Destruktor
        ~SivitasAkademik(){}
};